client/components/mma/cancel/CancellationReasonReview.tsx

```diff
@@ -287,8 +287,13 @@ const ConfirmCancellationAndReturnRow = (
 	const isMonthlyBilling =
 		isPaidSubscriptionPlan(mainPlan) && mainPlan.billingPeriod === 'month';
 
+	const allowCountrySwitchDiscount =
+		isPaidSubscriptionPlan(mainPlan) &&
```

Comment:
why do we need to check isPaidSubscriptionPlan everywhere?  is it just ruling out the old Friend/Staff?

Reply:
we don't! good spot, that was there when I thought that the billingCountry was a property on the main plan object which is related to paid subscription plans